layouts/authors/list.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html lang="{{ .Site.LanguageCode }}">
+
+  {{ partial "head.html" . }}
+
+  <body>
+
+    <div id="all">
+
+        {{ partial "nav.html" . }}
+
+        {{ partial "breadcrumbs.html" . }}
+
+        <div id="content">
+            <div class="container">
+                <div class="row">
+                    <!-- *** LEFT COLUMN *** -->
+
+                    <div class="col-md-9" id="blog-listing-medium">
+
+                        <section class="post">
```

Comment:
perhaps is better to use the same structure as posts:

https://github.com/devcows/hugo-universal-theme/blob/master/layouts/_default/list.html#L19-L85

So we will have the same behaviour like tags or categories. In the future perhaps we can add support to author bios.

Reply:
That makes it easier to implement ;-)